mono/mini/mini-llvm.c

```diff
@@ -9253,8 +9323,12 @@ mono_llvm_create_aot_module (MonoAssembly *assembly, const char *global_prefix,
 	module->plt_entries = g_hash_table_new (g_str_hash, g_str_equal);
 	module->plt_entries_ji = g_hash_table_new (NULL, NULL);
 	module->direct_callables = g_hash_table_new (g_str_hash, g_str_equal);
-	module->method_to_lmethod = g_hash_table_new (NULL, NULL);
+	module->module_nonnull = g_hash_table_new (NULL, NULL);
 	module->idx_to_lmethod = g_hash_table_new (NULL, NULL);
+	module->method_to_lmethod = g_hash_table_new (NULL, NULL);
+	module->lmethod_to_method = g_hash_table_new (NULL, NULL);
+	module->method_to_call_info = g_hash_table_new (NULL, NULL);
```

Comment:
Do we care, particularly that this hash table (and the arrays in its values) are never freed?

Reply:
It looks like we're never freeing any of these hash tables. It's good not to leak a lot of memory on exit, but the lifetime of these are such that they would be freed right before shutdown.